view.go

```diff
@@ -46,6 +53,12 @@ func init() {
 }
 
 func renderStyles() {
+	defer func() {
+		if rec := recover(); rec != nil {
+			fmt.Fprintf(os.Stderr, "An error occurred during renderStyles: %v\n", rec)
+		}
+	}()
```

Comment:
I like this catch-all. Would it possible to add some text that directs the user to raise an issue in Github? Maybe a pre-populated link?